boards/arm/thingy91_nrf9160/thingy91_pm_static.yml

```diff
@@ -1,4 +1,4 @@
-app: {address: 0x18200, size: 0x5ae00}
```

Comment:
We are unsure if this might break backwards compatibility with existing thingys. @jorgenmk will do a test of it asap

Reply:
Our thought was as the mcuboot primary partition address doesn't change, it shouldn't. 
TF-M and the application are getting updated together so the change of the adresse between TF-M and an non-secure app shouldn't matter as i looks the like the same for mcuboot.

Reply:
This is verified working for old Thingy91 releases.